ADD_BLOG_ENTRY.md

```diff
@@ -0,0 +1,76 @@
+## Adding a blog entry
+
+Blog entries can be added by following these steps:
+
+- If submitting for the first time an [author](#blog-author) YAML file must be added to the `data/blog/authors` directory.
+- Add an MD file containing the [blog post's metadata and body](#blog-post-content).
+  - each file must be added in the current year's directory, found in `data/blog/`.
+
```

Comment:
I suggest that you state the filename convention yyyy-mm-dd-title with an example